### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by iljaSL**
Included link to API docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ The web application is for education purpose only and is not meant for streaming
 
 - [Team](#team)
 - [Tech stack](#tech-stack)
+- [API Documentation](https://app.swaggerhub.com/apis/hypertubeHive/Hypertube/1.0.0)
 - [Functionality](#functionality)
 - [App Video Preview](#app-live-preview)
 - [Preview on Heroku(TBC)](https://hypertube-demo.herokuapp.com/)
```
Comment (DianaMukaliyeva) [inline README.md]: By the way, could you please delete `(TBC)`. I forgot to do that :)

File: README.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ The web application is for education purpose only and is not meant for streaming
 
 - [Team](#team)
 - [Tech stack](#tech-stack)
+- [API Documentation](https://app.swaggerhub.com/apis/hypertubeHive/Hypertube/1.0.0)
```
Comment (DianaMukaliyeva) [inline README.md]: Good job!
maybe use this link? https://app.swaggerhub.com/apis-docs/hypertubeHive/Hypertube/1.0.0 to see the docs, without the possibility to edit.

**Review by DianaMukaliyeva — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (iljaSL) [reply README.md]: good point! changed it 👍

**Review by DianaMukaliyeva — APPROVED** [review]
Thank you!

**Review by T7Q — APPROVED** [review]
looks good

**Pull request merged**